DOCS-3519 Modifying Data and Context Edit (#14065)
DataDog/documentation: content/en/real_user_monitoring/browser/modifying_data_and_context.md

## Override default RUM view names

The RUM SDK automatically generates a [view event][2] for each new page visited by your users, or when the page URL is changed (for single page applications). A view name is computed from the current page URL, where variable alphanumeric IDs are removed automatically. For example, "/dashboard/1234" becomes "/dashboard/?".

Starting with [version 2.17.0][3], you may specify your own view names by tracking view events manually with the `trackViewsManually` option:

1. Set `trackViewsManually` to true when initializing RUM.

> 2. 

Author: ```suggestion
```

{{% /tab %}}
{{< /tabs >}}

If you are using React, Angular, Vue or any other frontend framework, Datadog recommends implementing the `startView` logic at the framework router level.

## Enrich and control RUM data

> The RUM Browser SDK captures RUM events and populates their main attributes. The `beforeSend` callback function gives you access to every event collected by the RUM SDK before they are sent to Datadog. 

Reviewer: ```suggestion
The RUM Browser SDK captures RUM events and populates their main attributes. The `beforeSend` callback function gives you access to every event collected by the RUM SDK before it is sent to Datadog. 
```
